src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -337,8 +354,27 @@ public void deleteUsers(String query, RoutingContext routingContext,
       Handler<AsyncResult<Response>> asyncResultHandler,
       Context vertxContext) {
 
-    PgUtil.delete(getTableName(null), query, okapiHeaders, vertxContext,
-        DeleteUsersResponse.class, asyncResultHandler);
+    try {
+      CQLWrapper wrapper = getCQL(query, -1, -1);
```

Comment:
The same values of offset/limit as previous implementation is doing
![image](https://user-images.githubusercontent.com/25097693/233684587-501f169f-c389-48b2-b024-4e676c678a47.png)